plugins/modules/eos_config.py

```diff
@@ -226,6 +226,12 @@
           in C(filename) within I(backup) directory.
         type: path
     type: dict
+  timer:
+    description:
+    - This argument will configure a value in times for the commit to be confirmed 
+      before it is automatically rolled back. Value is define as 1h2m3s and will be
+      converted on the switches using the arista format 01:02:03.
```

Comment:
Worth giving example to demonstrate optional fields, eg "90s" or "1m30s"

Reply:
they put example below the description, i put one for timer like this
    - name: adding a commit timer
      arista.eos.eos_config:
        timer: 1m30s